pkg/mounter/proxy/server/ossfs2/driver.go

```diff
@@ -142,6 +164,11 @@ func (h *Driver) Mount(ctx context.Context, req *proxy.MountRequest) error {
 	})
 
 	if err == nil {
+		if monitor != nil {
+			monitor.HandleMountSuccess(cmd)
+			// Start monitoring goroutine (ticker based only)
+			h.monitorManager.StartMonitoring(target)
```

Comment:
Can we merge `StartMonitoring` into `HandleMountSuccess`? They seems always be called together.